src/sentry/rules/filters/latest_adopted_release_filter.py

```diff
@@ -28,57 +29,58 @@
 
 
 class LatestAdoptedReleaseForm(forms.Form):
-    release_age_type = forms.ChoiceField(choices=list(model_age_choices))
-    age_comparison = forms.ChoiceField(choices=list(age_comparison_choices))
-    environment_id = forms.CharField(required=False)
+    oldest_or_newest = forms.ChoiceField(choices=list(model_age_choices))
+    older_or_newer = forms.ChoiceField(choices=list(age_comparison_choices))
+    environment = forms.CharField(required=False)
```

Comment:
it's marked as`required=False`, but I don't see any default. Wouldn't not setting it just default to `""` and then raise  `forms.ValidationError( "environment does not exist...`?

Reply:
Ahh it's required, just an oversight there